project.yaml

```diff
@@ -116,56 +117,56 @@ targets:
       x86_64:
         test_distros:
           representative:
-          - rocky-linux-9
+          - rocky-linux-cloud:rocky-linux-9
           exhaustive:
-          - rhel-9
+          - rhel-cloud:rhel-9
       aarch64:
         test_distros:
           representative:
-          - rocky-linux-9-arm64
+          - rocky-linux-cloud:rocky-linux-9-arm64
           exhaustive:
-          - rhel-9-arm64
+          - rhel-cloud:rhel-9-arm64
   sles12:
     package_extension:
       rpm
     architectures:
       x86_64:
         test_distros:
           representative:
-          - sles-12
+          - suse-cloud:sles-12
           exhaustive:
-          - sles-12-sp5-sap
+          - suse-sap-cloud:sles-12-sp5-sap
   sles15:
     package_extension:
       rpm
     architectures:
       x86_64:
         test_distros:
           representative:
-          - sles-15
+          - suse-cloud:sles-15
           exhaustive:
-          - sles-15-sp2-sap
-          - sles-15-sp5-sap
-          - opensuse-leap
-          - opensuse-leap-15-4
+          - suse-sap-cloud:sles-15-sp2-sap
+          - suse-sap-cloud:sles-15-sp5-sap
+          - opensuse-cloud:opensuse-leap
+          - opensuse-cloud=opensuse-leap-15-4-v20231208-x86-64
```

Comment:
Why? this seems correct to me.

Reply:
why is this PR changing this line though? https://screenshot.googleplex.com/5Ri285yfB7HuUKj

Reply:
because that's the image getting tested, it's overridden here: https://github.com/GoogleCloudPlatform/ops-agent/blob/master/integration_test/gce/gce_testing.go#L1028